Package.resolved

```diff
@@ -1,13 +1,58 @@
 {
```

Comment:
Why is this file checked into version control?

Reply:
I'll admit I had the same thought.  Let's delete it